dts/bindings/mtd/partition.yaml

```diff
@@ -16,6 +16,17 @@ properties:
 child-binding:
     description: Flash partition child node
     properties:
+       mntauto:
```

Comment:
`automount` is fairly standard terminology that I think would be more clear.

Reply:
Yeah, will change.